deepface/modules/recognition.py

```diff
@@ -145,25 +151,36 @@ def find(
     # Get the list of images on storage
     storage_images = image_utils.list_images(path=db_path)
 
-    if len(storage_images) == 0:
```

Comment:
this control should not be dropped. you may want to add refresh_database check as well.

i mean `if len(storage_images) == 0 and refresh_database is True`